### PR TITLE
"Archive" unnecessary fields

### LINKED ISSUE #908: Resolve public vs workflow field
We currently only allow `public` policies to be accessed in the document endpoint. We recently added a workflow phase field which describes the _document_'s phase of being validation.

Do we replace the `public` field with that document workflow? Which policies are we listing and which load in the document view? How do we ensure that logged in users _do_ have access to edit documents?

### PR DESCRIPTION
**Pull request by cmc333333**
Per @rtwell and @austinhernandez's [analysis](https://github.com/18F/omb-eregs/issues/855), a large number of fields we imported from the original data set are no longer helpful. This keeps imported data, but hides a swath of fields from the admin, api, and ui.

This keeps `public` (until we resolve #908) and `workflow_phase` (until we implement automated transitions). It also does nothing about requirements, though we'll likely want to archive them soon.

## After
screen shot 2018-01-25 at 3 00 48 pm

### CODE REVIEW

File: api/ereqs_admin/tests/admin_tests.py [hunk 1]
```diff
@@ -85,6 +85,7 @@ def test_user_edit_max(admin_client):
     assert user.first_name == 'posttest'
 
 
+@pytest.mark.skip(reason="We've temporarily removed pdf upload")
```
Comment (toolness) [inline api/ereqs_admin/tests/admin_tests.py]: Wait, we _supported_ pdf upload?! 😂

Comment (tadhg-ohiggins) [reply api/ereqs_admin/tests/admin_tests.py]: Heh, indeed, although just in the file sense, without any import-and-convert aspect.

File: api/reqs/models.py [hunk 2]
```diff
@@ -108,30 +108,32 @@ def __str__(self):
...
-    )
     slug = models.SlugField(max_length=title.max_length)
     issuance = models.DateField()
     sunset = models.DateField(blank=True, null=True)
-    policy_status = models.CharField(max_length=256, blank=True)
-    document_source = models.FileField(blank=True)
     public = models.BooleanField(default=True)
-    issuing_body = models.CharField(max_length=512)
-    managing_offices = models.ManyToManyField(
-        Office, blank=True, related_name='policies')
```
Comment (tadhg-ohiggins) [inline api/reqs/models.py]: I have this feeling that a bunch of the above are going to come back, but I can see the argument for removing them now.

Comment (cmc333333) [reply api/reqs/models.py]: Agreed! I suspect they might also come back on a different form.

**Comment by cmc333333** [comment]
Thanks @toolness and @tadhg-ohiggins!

**Pull request merged**